dev_tools/scripts/deploy-mq.sh

```diff
@@ -5,6 +5,7 @@ CEGA_MQ_PASS=$(grep cega_mq_pass sda-deploy-init/config/trace.yml | awk '{print
 
 helm install broker sda-mq \
 --set securityPolicy.create=false,\
+config.verifyPeer=false,\
```

Comment:
I think this is fine to merge and get tests enabled, but we really should fix the problem as well.

Reply:
Agreed